src/network/core/host.cpp

```diff
@@ -200,6 +200,6 @@ void NetworkFindBroadcastIPs(NetworkAddressList *broadcast)
 	int i = 0;
 	for (NetworkAddress &addr : *broadcast) {
 		addr.SetPort(NETWORK_DEFAULT_PORT);
-		DEBUG(net, 3, "%d) %s", i++, addr.GetHostname());
+		DEBUG(net, 3, " %d) %s", i++, addr.GetHostname());
```

Comment:
The other places (in network) where there's whitespace at the begin of the sentence, there are two spaces. This is the only place that would introduce a single space.